### ISSUE TITLE
Questions: Multiple query, Condition grouping, Count?

### ISSUE THREAD

**Issue by AlexandreRoba** [jakin010/surrealdb_extra#3]
Hi thanks for this crates. It looks promising.

 I need to build such a query:
```
        select * from organisation where partner=$partner AND (
            owners contains $user or
            partner.owners contains $user or
            partner.member contains $user) limit by $limit_by start at $start_at;
        return count(select * from organisation where partner=$partner AND (
            owners contains $user or
            partner.owners contains $user or
            partner.member contains $user
        )); 
```

I have couple of questions:

### Combine multiple queries

Can you include more than one query on one builder?

### How do multiple `condition` combine? 
Is this:
```
.condition(("partner", Operator::Equal, "$partner"))
            .condition(cond_vec!(
                ("owners", Operator::Contain, "$user"),
                Operator::Or,
                ("partner.owners", Operator::Contain, "$user"),
                Operator::Or,
                ("partner.members", Operator::Contain, "$user"),
            ))
```
equivalent to `partner = $partner AND ( owners CONTAINS $user OR .... OR....)` ?

### Can you do a Count(SELECT....)?

**Comment by jakin010** [comment]
Right now there is no support for multiple queries. Currently more focused on creating the builder for the other statements like CREATE, INSERT and more.

As for multiple conditions if you use conditions twice the second one is going to override the first condition function. I'm going to make it impossible to use the 2 conditions in 1 builder in the future.

The statement you have in your example is possible to be made if you use the `Value` type that is inside surrealdb. Personally i know that is possible but without going deep in to it no clue how to write it now

**Comment by jakin010** [comment]
Just made recursive condition available 0.6.0 was surprisingly easy

Count and other types of that sort will come at a later stage
